packages/terra-horizontal-divider/tests/jest/__snapshots__/HorizontalDivider.test.jsx.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
These snapshots need to be updated. That's why your build is failing.

Reply:
Updated https://github.com/cerner/terra-core/commit/9f8a5deba40296bf91341b6ff1d0d56f4bae16ac#diff-46429ad4c25a0b00f25c0054ce0dddf6R24